### PR TITLE
Create BurgerClaimerSponsoredByFlamingo.yml

### PR DESCRIPTION
**Pull request by vang1ong7ang**

### CODE REVIEW

**Comment by Hecate2** [comment]
Sure to run `cron: "0 * * * *"`, that is every minute?

File: .github/workflows/BurgerClaimerSponsoredByFlamingo.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - cron: "0 * * * *"
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.x
```
Comment (Hecate2) [inline .github/workflows/BurgerClaimerSponsoredByFlamingo.yml]: 7.0.x

File: .github/workflows/BurgerClaimerSponsoredByFlamingo.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 7.0.x
+    - name: Restore dependencies
+      run: dotnet restore
+    - name: Run Claimer
+      run: echo ${{ secrets.CONFIG_MAINNET }} | dotnet run --project BurgerClaimer
+      env:
+        WIF: ${{ secrets.TEEFLMWIF }}
+        RPC: https://n3seed2.ngd.network:10332
```
Comment (dusmart) [inline .github/workflows/BurgerClaimerSponsoredByFlamingo.yml:22]: we need to add a env `MOD: 3` here to trigger only 1/3 here.

**Review by dusmart — APPROVED** [review]

**Review by Hecate2 — APPROVED** [review]

**Comment by vang1ong7ang** [comment]
@dusmart is it okay?

**Pull request merged**